sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelHealthChecker.java

```diff
@@ -104,6 +115,7 @@ public RntbdClientChannelHealthChecker(final Config config) {
         this.timeoutOnWriteTimeLimitInNanos = config.timeoutDetectionOnWriteTimeLimitInNanos();
         this.nonRespondingChannelReadDelayTimeLimitInNanos = config.nonRespondingChannelReadDelayTimeLimitInNanos();
         this.cancellationCountSinceLastReadThreshold = config.cancellationCountSinceLastReadThreshold();
+        this.clientTelemetry = clientTelemetry;
```

Comment:
this line should have resulted in compiler error - this. isn't allowed for static field.